### DISCUSSION TITLE
Can't able to generate QR Code  missing or malformed jwt

### DISCUSSION THREAD

**Discussion by pandiarajanvp** [dimaskiddo/go-whatsapp-multidevice-rest#65] [Q&A]
Request:

curl -X 'POST' \ 'http://127.0.0.1:3000/api/v1/whatsapp/login' \ -H 'accept: application/json' \ -H 'Authorization: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXQiOnsiamlkIjoiYWRtaW4ifSwiaWF0IjoxNzE1NzYwMTU3fQ.UrU3_lOcYb5qsb0bpiIV8sxrzawTP7FyvEAj3H4I_2U' \ -H 'Content-Type: multipart/form-data' \ -F 'output=json'

Response 
Status Code 400

{
  "status": false,
  "code": 400,
  "error": "missing or malformed jwt"
}

**Comment by ibnux** [comment]
you forgot Bearer
